projects/canopy/src/lib/modal/modal.component.spec.ts

```diff
@@ -60,44 +60,60 @@ describe('LgModalComponent', () => {
   });
 
   describe('on init', () => {
-    it('should update isOpen', () => {
-      isModalOpen$.next(true);
+    describe('when the modal hasn\'t been opened/closed yet', () => {
+      it('should update isOpen', () => {
+        expect(component.isOpen).toBe(undefined);
+      });
+
+      it('should not emit a closed event', () => {
+        const closedEmitterSpy = spy(component.closed);
+
+        component.ngOnInit();
 
-      expect(component.isOpen).toBe(true);
+        verify(closedEmitterSpy.emit()).never();
+      });
     });
 
-    it('should add the overflow style to the body and emit an open event if the modal is open', () => {
-      const openEmitterSpy = spy(component.open);
+    describe('when the modal has been opened/closed yet', () => {
```

Comment:
```suggestion
    describe('when the modal has been already been opened/closed', () => {
```

Reply:
Updated to `when the modal has been opened/closed`.